templates/common/infra/bicep/core/database/sqlserver/sqlserver.bicep

```diff
@@ -46,7 +43,7 @@ resource sqlServer 'Microsoft.Sql/servers@2022-02-01-preview' = {
 }
 
 resource sqlDeploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
-  name: 'script-${resourceToken}'
+  name: '${name}-deployment-script'
```

Comment:
looks like a breaking change?  As the name is now different?

Reply:
Yes, there are a lot of breaking changes with this release, so I'm not concerned about this one. We don't currently have a way to "refresh" a template with latest version and guarantee it will work.